<ui version="4.0" >
 <class>SymbolManager</class>
 <widget class="QDialog" name="SymbolManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Symbol Manager</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSymFiles" >
      <attribute name="title" >
       <string>Symbol files</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="2" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Loaded symbol files:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QTreeWidget" name="treeFiles" >
         <property name="selectionMode" >
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="itemsExpandable" >
          <bool>false</bool>
         </property>
         <property name="sortingEnabled" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>File</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Destination slot</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Last refresh</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="1" >
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QPushButton" name="btnAddFile" >
           <property name="text" >
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemoveFile" >
           <property name="text" >
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAddrLabels" >
      <attribute name="title" >
       <string>Address labels</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Address labels:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QPushButton" name="btnAddSymbol" >
           <property name="text" >
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemoveSymbol" >
           <property name="text" >
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0" >
        <widget class="QTreeWidget" name="treeLabels" >
         <property name="selectionMode" >
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights" >
          <bool>true</bool>
         </property>
         <property name="itemsExpandable" >
          <bool>false</bool>
         </property>
         <property name="sortingEnabled" >
          <bool>true</bool>
         </property>
         <property name="allColumnsShowFocus" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>Symbol</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Type</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Value</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Slots</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Segments</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Registers</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Source</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QTabWidget" name="tabWidgetSymbols" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex" >
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabLocations" >
          <attribute name="title" >
           <string>Locations</string>
          </attribute>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QGroupBox" name="groupSlots" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title" >
              <string>Active slots</string>
             </property>
             <layout class="QHBoxLayout" >
              <item>
               <layout class="QVBoxLayout" name="vbox0" >
                <item>
                 <widget class="QCheckBox" name="chk00" >
                  <property name="text" >
                   <string>0-0</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="chk01" >
                  <property name="text" >
                   <string>0-1</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="chk02" >
                  <property name="text" >
                   <string>0-2</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="chk03" >
                  <property name="text" >
                   <string>0-3</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="vbox1" >
                <item>
                 <widget class="QCheckBox" name="chk10" >
                  <property name="text" >
                   <string>1-0</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="chk11" >
                  <property name="text" >
                   <string>1-1</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="chk12" >
                  <property name="text" >
                   <string>1-2</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="chk13" >
                  <property name="text" >
                   <string>1-3</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="vbox2" >
                <item>
                 <widget class="QCheckBox" name="chk20" >
                  <property name="text" >
                   <string>2-0</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="chk21" >
                  <property name="text" >
                   <string>2-1</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="chk22" >
                  <property name="text" >
                   <string>2-2</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="chk23" >
                  <property name="text" >
                   <string>2-3</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="vbox3" >
                <item>
                 <widget class="QCheckBox" name="chk30" >
                  <property name="text" >
                   <string>3-0</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="chk31" >
                  <property name="text" >
                   <string>3-1</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="chk32" >
                  <property name="text" >
                   <string>3-2</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="chk33" >
                  <property name="text" >
                   <string>3-3</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupSegments" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title" >
              <string>Active segments</string>
             </property>
             <layout class="QVBoxLayout" >
              <item>
               <widget class="QLineEdit" name="txtSegments" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabUsage" >
          <attribute name="title" >
           <string>Usage</string>
          </attribute>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QGroupBox" name="groupType" >
             <property name="title" >
              <string>Symbol type</string>
             </property>
             <layout class="QVBoxLayout" name="vboxSymbolType" >
              <item>
               <widget class="QRadioButton" name="radJump" >
                <property name="text" >
                 <string>Jump label</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radVar" >
                <property name="text" >
                 <string>Variable label</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radValue" >
                <property name="text" >
                 <string>Value</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupRegs8" >
             <property name="title" >
              <string>8 bit registers</string>
             </property>
             <layout class="QHBoxLayout" >
              <item>
               <layout class="QGridLayout" name="gridRegs8" >
                <property name="horizontalSpacing" >
                 <number>16</number>
                </property>
                <item row="0" column="0" >
                 <widget class="QCheckBox" name="chkRegA" >
                  <property name="text" >
                   <string>A</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1" >
                 <widget class="QCheckBox" name="chkRegI" >
                  <property name="text" >
                   <string>I</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2" colspan="2" >
                 <widget class="QCheckBox" name="chkRegOffset" >
                  <property name="text" >
                   <string>Offset value</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" >
                 <widget class="QCheckBox" name="chkRegB" >
                  <property name="text" >
                   <string>B</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1" >
                 <widget class="QCheckBox" name="chkRegC" >
                  <property name="text" >
                   <string>C</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0" >
                 <widget class="QCheckBox" name="chkRegD" >
                  <property name="text" >
                   <string>D</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1" >
                 <widget class="QCheckBox" name="chkRegE" >
                  <property name="text" >
                   <string>E</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2" >
                 <widget class="QCheckBox" name="chkRegIXH" >
                  <property name="text" >
                   <string>IXH</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="3" >
                 <widget class="QCheckBox" name="chkRegIXL" >
                  <property name="text" >
                   <string>IXL</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0" >
                 <widget class="QCheckBox" name="chkRegH" >
                  <property name="text" >
                   <string>H</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1" >
                 <widget class="QCheckBox" name="chkRegL" >
                  <property name="text" >
                   <string>L</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="2" >
                 <widget class="QCheckBox" name="chkRegIYH" >
                  <property name="text" >
                   <string>IYH</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="3" >
                 <widget class="QCheckBox" name="chkRegIYL" >
                  <property name="text" >
                   <string>IYL</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupRegs16" >
             <property name="title" >
              <string>16 bit registers</string>
             </property>
             <layout class="QHBoxLayout" >
              <item>
               <layout class="QGridLayout" name="gridRegs16" >
                <item row="0" column="0" >
                 <widget class="QCheckBox" name="chkRegBC" >
                  <property name="text" >
                   <string>BC</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" >
                 <widget class="QCheckBox" name="chkRegDE" >
                  <property name="text" >
                   <string>DE</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0" >
                 <widget class="QCheckBox" name="chkRegHL" >
                  <property name="text" >
                   <string>HL</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0" >
                 <widget class="QCheckBox" name="chkRegIX" >
                  <property name="text" >
                   <string>IX</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1" >
                 <widget class="QCheckBox" name="chkRegIY" >
                  <property name="text" >
                   <string>IY</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnReloadFiles" >
       <property name="text" >
        <string>Reload all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>SymbolManager</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>571</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel" >
     <x>233</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
